spec/features/account/backup_codes_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+feature 'Backup codes' do
+  before do
+    sign_in_and_2fa_user(user)
+  end
+
+  context 'with backup codes' do
+    let(:user) { create(:user, :with_backup_code) }
+
+    it 'shows backup code section' do
+      expect(page).to have_xpath("//div/em[starts-with(text(),'generated')]")
```

Comment:
It may be a touch more readable to replace the XPaths here with something like this:

```ruby
expect(page).to have_content(t('account.backup_codes.generated'))
```